src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dApp: DApp;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	{#if dApp.screenshots && dApp.screenshots.length > 0}
```

Comment:
A `nonNullish` is missing here (we are using this as convention for null check)

Reply:
thanks